internal/app/machined/internal/phase/acpi/acpi.go

```diff
@@ -9,14 +9,23 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"strings"
 
+	"github.com/hashicorp/go-multierror"
 	"github.com/mdlayher/genetlink"
+	"github.com/mdlayher/netlink"
 
 	"github.com/talos-systems/talos/internal/app/machined/internal/phase"
 	"github.com/talos-systems/talos/internal/pkg/event"
 	"github.com/talos-systems/talos/internal/pkg/runtime"
 )
 
+const (
+	// PowerButtonEvent is the ACPI event name associated with the power off
+	// button.
+	PowerButtonEvent = "button/power"
```

Comment:
Is button/power the only one we need to handle?

Reply:
As far as I could tell. The docs on ACPI events are practically non-existent, or just hard to find.